db/import_column_family_job.cc

```diff
@@ -97,6 +97,10 @@ Status ImportColumnFamilyJob::Prepare(uint64_t next_file_number,
       if (status.IsNotSupported()) {
         // Original file is on a different FS, use copy instead of hard linking
         hardlink_files = false;
+        ROCKS_LOG_INFO(
+            db_options_.info_log,
+            "AddFile() tries to link file %s but it's not supported : %s",
```

Comment:
Should we also add logging to `ExternalSstFileIngestionJob::Prepare()`?

Reply:
Good catch... I modified in the wrong place... Fixed it.